eng/Packages.Data.props

```diff
@@ -259,6 +259,7 @@
     <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="2.1.10" />
     <PackageReference Update="Microsoft.Extensions.Configuration.Json" Version="5.0.0" />
     <PackageReference Update="Microsoft.Extensions.Configuration" Version="5.0.0" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Configuration" Version="8.0.0-rc.1.23419.4" />
```

Comment:
Why do we need this reference?

Reply:
It's needed for full analysis. We need to reference it in AOT test projects, so I figured we should manage it centrally like any other package.

Reply:
Yeah it gets the most up-to-date trimming warnings so that the AOT-compatibility tests can be more accurate.